custom-words.txt

```diff
@@ -588,15 +593,15 @@ environmentsettings
 EPSG
 errordetail
 errored
-ErrorMesssage
```

Comment:
Why removed these two strings?

Reply:
CamelCase words are already split by cSpell, so this wasn't needed